services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceParentHelixAdmin.java

```diff
@@ -1227,6 +1229,22 @@ Optional<String> getTopicForCurrentPushJob(
           }
           return Optional.empty();
         } else {
+          Map<String, ControllerClient> controllerClients = getVeniceHelixAdmin().getControllerClientMap(clusterName);
+          for (Map.Entry<String, ControllerClient> entry: controllerClients.entrySet()) {
+            ControllerClient controllerClient = entry.getValue();
+            StoreResponse response = controllerClient.getStore(storeName);
```

Comment:
Just in case, shall we use the retry API here? ControllerClient#retryableRequest with 3 attempts?